src/types/BaseField.ts

```diff
@@ -101,8 +107,19 @@ const writableBaseFieldSchema: JSONSchemaType<WritableBaseField> = {
 			type: 'string',
 			enum: Object.values(BaseFieldScope),
 		},
+		valueRelevanceHours: {
+			type: 'number',
+			nullable: true as false, // see https://github.com/ajv-validator/ajv/issues/2163
+		},
```

Comment:
Do we want to limit it to positive integers here?

Reply:
Yeah very good idea ty

Reply:
I decided to allow 0 as well just because who am I to not think of a situation where you wanna store data and immediately consider it irrelevant.